### ISSUE TITLE
AttributeError: 'PyTestServiceClass' object has no attribute 'issue_types' when run pytest without `--reportportal` and test fails

### ISSUE THREAD

**Issue by itepliuk** [reportportal/agent-python-pytest#140]
AttributeError is received when run pytest  **without** marker `--reportportal` and **test** from suite **fails** with any`Exception`, it breaks whole test run,
no matter what is the version of pytest(used pytest==5.1.2):
project: platform darwin -- Python 3.7.4, pytest-5.1.2, py-1.8.0, pluggy-0.13.0
`pytest-reportportal==1.0.8
reportportal-client==3.2.3
`
```
INTERNALERROR> Traceback (most recent call last):
......
INTERNALERROR>   File "/Users/#####/code/penv/lib/python3.7/site-packages/pytest_reportportal/listener.py", line 78, in pytest_runtest_makereport
INTERNALERROR>     self._add_issue_info(item, report)
INTERNALERROR>   File "/Users/#####/code/penv/lib/python3.7/site-packages/pytest_reportportal/listener.py", line 115, in _add_issue_info
INTERNALERROR>     if issue_type and self.PyTestService.issue_types \
INTERNALERROR> AttributeError: 'PyTestServiceClass' object has no attribute 'issue_types'
```
> - Is there a workaround?

**Comment by iivanou** [comment]
@itepliuk Can you apply fixes from #141 and try again?

**Comment by itepliuk** [comment]
Solved. Thanks for the fix, applied and now that works for me.
